lib/Cron/SocialUpdateRegistration.php

```diff
@@ -4,6 +4,7 @@
 
 /**
  * @copyright 2017 Georg Ehrke <oc.list@georgehrke.com>
+ * @copyright Copyright (c) 2022 Informatyka Boguslawski sp. z o.o. sp.k., http://www.ib.pl/
```

Comment:
triple copyright and no https? :stuck_out_tongue:

Reply:
https://github.com/nextcloud/server/blob/master/contribute/HowToApplyALicense.md